contributions/contribution.yaml

```diff
@@ -391,6 +391,80 @@ paths:
           description: Bad request
         500:
           description: Internal server error
+  /contributions/admin/reviewers:
```

Comment:
I'm suggesting a few changes to the YAML file. Please see:


```
diff --git a/contributions/contribution.yaml b/contributions/contribution.yaml
index ddb47df..b9e78e2 100644
--- a/contributions/contribution.yaml
+++ b/contributions/contribution.yaml
@@ -12,7 +12,13 @@ servers:
   description: Development server
 tags:
 - name: Contributions
-  description: API endpoints for managing contributions in the form of Talents and Capabilities.
+  description: API endpoints for managing Contributions.
+- name: Talents
+  description: API endpoints for managing Talents.
+- name: Capabilities
+  description: API endpoints for managing Capabilities.
+- name: Admin
+  description: Admin endpoints for the Contributions Building Block.
 paths:
   /contributions:
     post:
@@ -395,7 +401,7 @@ paths:
     post:
       summary: Create a new reviwer
       tags:
-        - Contributions
+        - Admin
       security:
         - GitHubAuth: [ ]
       requestBody:
@@ -424,7 +430,7 @@ paths:
     get:
       summary: Return the list of existing reviewers
       tags:
-        - Contributions
+        - Admin
       security:
         - GitHubAuth: [ ]
       responses:
@@ -446,7 +452,7 @@ paths:
     delete:
       summary: Delete a reviewer
       tags:
-        - Contributions
+        - Admin
       security:
         - GitHubAuth: [ ]
       parameters:

```

Reply:
I am not very sure if I understand correctly, should I change the things in here? For example, 
'description: API endpoints for managing contributions in the form of Talents and Capabilities.' 
should be changed like
'description: API endpoints for managing Contributions.` ?

Reply:
Yes, that's right. I wanted to make a few suggestions, hence I shared the diff instead of individual suggestions.

Reply:
Updated